### DISCUSSION TITLE
overrides.cfg file

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#787] [Ideas]
### What feature would you like to see?

It'd be awesome to have something like Librewolf's `librewolf.overrides.cfg`, a JS file that will be loaded at startup where one can set values for `about:config` flags declaratively. It would make it easier to set up Zen on new machines, as the file can be easily shared across devices.

### Other details

Librewolf's docs on how this works: https://librewolf.net/docs/settings/
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/599_

**Comment by Bammede** [comment]
When overrides.cfg upload feature will be available?

**Comment by robertoandrade** [comment]
also looking for something like this or at least command line arguments that can be passed on first launch to setup the flags we'd like, ie: opening the URI with parameters:

```
zen about:config?config1=value&config2=value
```
or
```
zen about:config#config1=value&config2=value
```
as hash references could be used to link people directly to a given flag if we'd like to point them from forums or documentation. Perhaps the querystring could be used for setting it while the hash could be used for simply linking them to the exact config as if the user was searching for it.

Additionally, this should skip that initial warning about accepting the risks of messing with configuration, especially if this is to be used as a way to launch zen in automation scripts to setup the browser for users using a specific set of configs. Again as an alternative to the librewolf approach which I agree is preferrable.
